script.js

```diff
@@ -1,4 +1,249 @@
+      // By default, the card name is the same as rankCounter
+      var cardName = rankCounter;
+      var cardValue = valueCounter;
+      var cardValueAce1 = valueCounter;
+      var cardValueAce11 = valueCounter;
+​
+      // If rank is 1, 11, 12, or 13, set cardName to the ace or face card's name
+      if (cardName == 1) {
+        cardName = "ace";
+        cardValue = 1;
+      } else if (cardName == 11) {
+        cardName = "jack";
+        cardValue = 11;
+      } else if (cardName == 12) {
+        cardName = "queen";
+        cardValue = 12;
+      } else if (cardName == 13) {
+        cardName = "king";
+        cardValue = 13;
+      }
+​
+      // Create a new card with the current name, suit, and rank
+      var card = {
+        name: cardName,
+        suit: currentSuit,
+        rank: rankCounter,
+        value: cardValue,
+      };
+​
+      // Add the new card to the deck
+      cardDeck.push(card);
+​
+      // Increment rankCounter to iterate over the next rank
+      rankCounter += 1;
+    }
+​
+    // Increment the suit index to iterate over the next suit
+    suitIndex += 1;
+  }
+​
+  // Return the completed card deck
+  return cardDeck;
+};
+​
+var myOutputValue = `KNN CCB CAN U PUT IN THE RIGHT WORD`;
+​
+var getRandomIndex = function (max) {
+  return Math.floor(Math.random() * max);
+};
+var shuffleCards = function (cardDeck) {
+  index = 0;
+  while (index < cardDeck.length) {
+    var randomIndex = getRandomIndex(cardDeck.length);
+    var randomCard = cardDeck[randomIndex];
+    var currentCard = cardDeck[index];
+    cardDeck[index] = randomCard;
+    cardDeck[randomIndex] = currentCard;
+    index++;
+  }
+  return cardDeck;
+};
+​
+var deck = shuffleCards(makeDeck());
+​
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var myOutputValue = "hello world";
+  var myOutputValue = ``;
+  var player1stCard = deck.pop();
+  var computer1stCard = deck.pop();
+  var player2ndCard = deck.pop();
+  var computer2ndCard = deck.pop();
+  var playerDraw = deck.pop();
+  var computerDraw = deck.pop();
+  var playerDraw2 = deck.pop();
+  var computerDraw2 = deck.pop();
+  var playerCardValue = Number(player1stCard.rank) + Number(player2ndCard.rank);
+  var computerCardValue =
+    Number(computer1stCard.rank) + Number(computer2ndCard.rank);
+​
+  // Click submit to deal cards to both player and computer
+  var genericOutput = ` PLAYER CARDS: <br> 1ST PLAYER CARD: ${player1stCard.name} of ${player1stCard.suit} <br> 2ND PLAYER CARD: ${player2ndCard.name} of ${player2ndCard.suit}  <br>COMPUTER CARDS: <br> 1ST COM CARD: ${computer1stCard.name} of ${computer1stCard.suit} <br> 2ND COM CARD: ${computer2ndCard.name} of ${computer2ndCard.suit}`;
+​
+  var genericScores = `<br> PLAYER SCORE = ${playerSum}. Computer score = ${computerSum}`;
+​
+  if (currentGameMode == start) {
+    currentGameMode = "commence";
+  }
+  if (currentGameMode == `commence`) {
```

Comment:
If you do not have variables in your strings, use single inverted commas instead of `